libafl/src/inputs/mod.rs

```diff
@@ -53,7 +55,7 @@ pub trait Input: Clone + Serialize + serde::de::DeserializeOwned + Debug {
     }
 
     /// Generate a name for this input
-    fn generate_name(&self, idx: usize) -> String;
+    fn generate_name(&self, id: Option<CorpusId>) -> String;
```

Comment:
why this guy is wrapped in Option?

Reply:
Because we don't set it in many places / just pass 0